### PR TITLE
Invouce amounts rounded to 2 decimals

### PR DESCRIPTION
**Pull request by jeroenmac**
Updated the following: 
1) round tax amount to 2 decimals places
2) round subtotal amount to 2 decimal places
3) round total amount to 2 decimal places

### CODE REVIEW

**Comment by zhangshine** [comment]
@jeroenmac Hi, thanks for your PR.

Could use [Decimal.quantize](http://devdocs.io/python~2.7/library/decimal#decimal.Decimal.quantize) instead of `round` to round these values to 2 decimals? All tests have failed because `round` returns  different results under different Python versions.

Sorry for my bad english.

**Comment by jeroenmac** [comment]
Thank you for your feedback.
1. I've updated the function to use Decimal.quantize
2. I've added the precision parameter to the constructor

**Comment by zhangshine** [comment]
@jeroenmac Hi, could you update the test cases? just run `python setup test` and fix it.

This repo is integrate with travis ci, when the tests failed it will show a :x: icon on the right of your commit message. You can click the icon to see the detail.

File: pyinvoice/templates.py [hunk 1]
```diff
@@ -15,8 +15,9 @@
 
 class SimpleInvoice(SimpleDocTemplate):
     default_pdf_info = PDFInfo(title='Invoice', author='CiCiApp.com', subject='Invoice')
+    precision = None
 
-    def __init__(self, invoice_path, pdf_info=None):
+    def __init__(self, invoice_path, pdf_info=None,precision='0.01'):
```
Comment (zhangshine) [inline pyinvoice/templates.py]: Add a whitepsace between `,` and `precision`, check [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/) please.

File: pyinvoice/templates.py [hunk 2]
```diff
@@ -216,8 +219,9 @@ def _item_data_and_style(self):
         sum_start_x_index = len(item_data_title) - abs(sum_end_x_index)
 
         # ##### Subtotal #####
+        rounditem_subtotal = self.getroundeddecimal(item_subtotal,self.precision)
```
Comment (zhangshine) [inline pyinvoice/templates.py]: Same as above

File: pyinvoice/templates.py [hunk 3]
```diff
@@ -226,8 +230,9 @@ def _item_data_and_style(self):
         # Tax total
         if self._item_tax_rate is not None:
             tax_total = item_subtotal * (Decimal(str(self._item_tax_rate)) / Decimal('100'))
+            roundtax_total = self.getroundeddecimal(tax_total,self.precision)
```
Comment (zhangshine) [inline pyinvoice/templates.py]: Same as above

File: pyinvoice/templates.py [hunk 4]
```diff
@@ -237,13 +242,20 @@ def _item_data_and_style(self):
 
         # Total
         total = item_subtotal + (tax_total if tax_total else Decimal('0'))
-        item_data.append(('Total', '', '', '', total))
+        roundtotal = self.getroundeddecimal(total,self.precision)
```
Comment (zhangshine) [inline pyinvoice/templates.py]: And here, thanks

**Comment by jeroenmac** [comment]
Dear, 

I've updated the test cases. 

Kr, 
Jeroen

**Pull request merged**